lerna.json

```diff
@@ -2,7 +2,6 @@
   "packages": [
     "packages/*",
     "packages/samples/**",
-    "packages/atomic-angular/*",
```

Comment:
Not sure why it's needed since `"packages/*",` & `"packages/atomic-angular/projects/*"` are there